.circleci/config.yml

```diff
@@ -221,8 +245,8 @@ commands:
           command: |
             cd rust
 
-            TARBALL_PATH="/tmp/${CIRCLE_PROJECT_REPONAME}-$(uname)-standard.tar.gz"
-            RELEASE_NAME="${CIRCLE_PROJECT_REPONAME}-$(uname)-standard"
+            TARBALL_PATH="/tmp/${CIRCLE_PROJECT_REPONAME}-$(uname)-$(uname -m)-standard.tar.gz"
+            RELEASE_NAME="${CIRCLE_PROJECT_REPONAME}-$(uname)-$(uname -m)-standard"
```

Comment:
This will probably require a change in lotus. @vmx any ideas?

Reply:
It's possible to keep the name the same by splitting up the jobs in config.yml, but it is simpler and I think more descriptive to have architecture (aarch64, x86-64) in the name in the long run.

Reply:
I really don't know much about the Lotus FFI integration. But to me it makes sense to have things named including their architecture if we plan to publish those as well. Thanks for flagging @Stebalien, we'll talk with the Lotus folks about it.

Reply:
Right now, the lotus build process relies mainly on the `install-filcrypto` script in the root of this project (https://github.com/filecoin-project/filecoin-ffi/blob/master/install-filcrypto#L79), so as long as that still works (or is updated to reference the new names) I think the lotus process should continue to function. Based on my reading of the script, I think it just takes the release name and appends `.tar.gz` which it looks like is what you're doing here as well, so I don't think it should require any change.

Once unzipped, are the libraries still called the same names?
```
filcrypto.h
libfilcrypto.a
filcrypto.pc
```

Reply:
I just pushed a commit to update the release_name in install-filcrypto script to add the platform architecture to the tarball name.  Tested it out to verify it pulls down the binary and is able to build with pre-compiled source.